src/main/java/net/dancier/service/RecommendationsService.java

```diff
@@ -0,0 +1,11 @@
+package net.dancier.service;
+
+import net.dancier.domain.Recommendation;
+import net.dancier.domain.User;
+
+import java.util.List;
+
+public interface RecommendationsService {
```

Comment:
Doc, please.

Reply:
done

Reply:
OK, with the documentation on a method level? Could not think of anything really useful an the interface Level.

Reply:
I really would write something like "Service for accessing recommendations." but I know you would reject because no real new info ;-) It's kind of an esthetic issue for me at least. But although I would and will write such a comment I won't force you. :-)

Reply:
Added this kind of documentation ;-)
Anything left before you can approve ;-)